### PR TITLE
deposit on demand option added to configuring the node

### PR DESCRIPTION
**Pull request by Niks988**

### CODE REVIEW

File: source/Configuration-setup.rst [hunk 1]
```diff
@@ -55,6 +55,53 @@ nodes for node’s self identification.
...
+
+.. code:: json
+
+    {
+        "node_wallet": "your wallet address here",
+        "node_private_key": "your wallet's private key here",
+        "network": {
+            "hostname": "your external IP or domain name here",
+            "remoteWhitelist": [ "IP or host of the machine that is requesting the import", "127.0.0.1"],
+            "deposit_on_demand": false
```
Comment (kipliklotrika) [inline source/Configuration-setup.rst]: nope. should be outside the network section.

File: source/Configuration-setup.rst [hunk 2]
```diff
@@ -55,6 +55,53 @@ nodes for node’s self identification.
...
```
Comment (kipliklotrika) [inline source/Configuration-setup.rst]: outside network.

File: source/Configuration-setup.rst [hunk 3]
```diff
@@ -55,6 +55,53 @@ nodes for node’s self identification.
...
+        }
+    }
+
+
+**ENABLING** the **deposit_on_demand** option:
+
+- In order to enable the **deposit_on_deman** option please edit your **config.json** file, which can be found in your "**/ot-node/config/**" directory, by setting it to "**true**" as described below:
```
Comment (kipliklotrika) [inline source/Configuration-setup.rst]: This is not correct.

File: source/Configuration-setup.rst (hunk 1 above)
Comment (radomir-sebek) [reply source/Configuration-setup.rst]: +1

**Pull request merged**